packages/dds/tree/src/forest/reader.ts

```diff
@@ -0,0 +1,39 @@
+/*!
```

Comment:
This file seems like it could be renamed to cursor.

Reply:
Oversight on my part.  (Thank you.)